packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -47,6 +49,32 @@ interface TokensListProps {
 }
 
 const getTokensByTheme = (theme: ThemeVariants, props: TokensListProps): TokenCategory[] => {
+  const font = props.consoleTokens[0].node.tokens.find(ele => ele.categoryName === 'fonts');
```

Comment:
@TheSisb @SiTaggart Do y'all know where this data structure is defined so we don't have to do this abomination?

Reply:
So we load the design tokens for each theme into Gatsby graphQL server and get them as page data here https://raw.githubusercontent.com/twilio-labs/paste/master/packages/paste-website/src/pages/tokens/index.mdx